topi/tests/python/test_topi_conv2d_nchw.py

```diff
@@ -35,6 +33,8 @@ def check_device(device):
             return
```

Comment:
@tqchen Can we check whether AVX2/AVX512 enables by `ctx.exist`? so that I can test `llvm -mcpu=skylake-avx512` and `llvm -mcpu=core-avx2` here.

Reply:
unfortunately, we can't do it via ctx.exist, we will need new ways to query CPU features